src/components/button/readme.md

```diff
@@ -33,9 +33,9 @@ Material Design guidelines: https://material.io/design/components/buttons.html
 
 ### Used by
 
- - [materials-datepicker](../datepicker)
- - [materials-dropdown](../dropdown)
- - [materials-step](../stepper)
+ - [materials-datepicker](..\datepicker)
```

Comment:
Windows way ?? Every time we generate doc if we have different OS it will add changes ?

Reply:
i'll fix it, we have to find a way to keep consistent across OS